sdk/samples/azidentity/manual-tests/managed-identity/functions/README.md

```diff
@@ -0,0 +1,148 @@
+# testing azidentity in Azure Functions
+
+# prerequisite tools
+- Azure CLI
+- Azure Functions Core Tools 4.x
+
+# Azure resources
+This test requires instances of these Azure resources:
+- Azure Key Vault
+- Azure Managed Identity
+  - with secrets/set and secrets/delete permission for the Key Vault
+- Azure Storage account
+- Azure App Service Plan
+- Azure Function App x2
+  - one for system-assigned identity, one for user-assigned
+
+The rest of this section is a walkthrough of deploying these resources.
+
+## Set environment variables
+- RESOURCE_GROUP
+  - name of an Azure resource group
+  - must be unique in the Azure subscription
+  - e.g. 'identity-test-rg'
+- APP_SERVICE_PLAN
+  - name of an Azure App Service Plan
+- FUNCTION_APP_SYSTEM_ASSIGNED
+  - name of an Azure function app
+  - must be globally unique
+- FUNCTION_APP_USER_ASSIGNED
+  - name of an Azure function app
+  - must be globally unique
+- MANAGED_IDENTITY_NAME
+  - name of the user-assigned identity
+  - 3-128 alphanumeric characters
+  - must be unique in the resource group
+- STORAGE_ACCOUNT_NAME
+  - 3-24 alphanumeric characters
+  - must be globally unique (check it with `az storage account check-name`)
+- KEY_VAULT_NAME
+  - 3-24 alphanumeric characters
+  - must begin with a letter
+  - must be globally unique
+
+## resource group
+```sh
+az group create -n $RESOURCE_GROUP --location westus2
+```
+
+## Key Vault:
+```sh
+az keyvault create -g $RESOURCE_GROUP -n $KEY_VAULT_NAME --sku standard
+```
+
+## Storage account
+```sh
+az storage account create -g $RESOURCE_GROUP -n $STORAGE_ACCOUNT_NAME
+```
+
+## App Service Plan
+```sh
+az appservice plan create -g $RESOURCE_GROUP -n $APP_SERVICE_PLAN -l westus2 --sku B1 --is-linux
+```
+
+## Functions App: system-assigned identity
+```sh
+az functionapp create -g $RESOURCE_GROUP -n $FUNCTION_APP_SYSTEM_ASSIGNED -s $STORAGE_ACCOUNT_NAME -p $APP_SERVICE_PLAN --runtime python
```

Comment:
I was looking at doing this in arm template/bicep the other day - it might shorten your instructions list at some point if you want to try that route.

Reply:
That would definitely be an improvement, but the path of least resistance was to copy this document from the Python repo and again punt on process improvement until we have infrastructure to automate these tests (https://github.com/Azure/azure-sdk/issues/899). Thanks for commenting on this line, it needs an edit because we're not deploying Python here...

Reply:
(resolve at your convenience)